### DISCUSSION TITLE
MobX Chrome Devtools don't show the "Components" tab

### DISCUSSION THREAD

**Discussion by mgcolossus** [mobxjs/mobx#2801] [Q&A] [answered]
Hi, Here's an image from chrome webstore
![image](https://user-images.githubusercontent.com/55920797/107854194-627de900-6e2b-11eb-9d8f-8e3d8922dee9.png)

I wonder why MobX Chrome Devtools don't show the "Components" tab although I've wrapped my components into observable() HOC
![image](https://user-images.githubusercontent.com/55920797/107854159-23e82e80-6e2b-11eb-928c-4fa553ea9ea8.png)

**Comment by iChenLei** [comment, marked as answer]
duplicated https://github.com/mobxjs/mobx-devtools/issues/56
![2021-02-17 10 57 26](https://user-images.githubusercontent.com/14012511/108150162-2b4f5680-710f-11eb-9912-00f671c3bd95.png)
